src/test/java/com/google/api/codegen/testdata/discoveries/sheets.v4.json.overrides

```diff
@@ -1,4 +1,15 @@
 {
+  "java": {
+    "methods": {
+      "sheets.spreadsheets.sheets.copyTo": {
+        "requestType": {
+          "message": {
+            "typeName": "Spreadsheets.SheetsOperations.CopyTo"
```

Comment:
The default name generated is `Sheets.Spreadsheets.Sheets.CopyTo`, which is wrong for whatever reason... I haven't had time to figure out the rule for this case. The actual type should be `Sheets.Spreadsheets.SheetsOperations.CopyTo`